frontend/app/professor/simulation-builder/page.tsx

```
@@ -2385,43 +2387,46 @@ return (
+                      // Use the new progress tracking for Business Case Study
+                      if (uploadedFile) {
+                        handleAutofillWithProgress();
+                      } else if (teachingNotesFile) {
+                        handleAutofillWithTeachingNotes();
+                      } else {
+                        console.log("No files uploaded for autofill");
+                      }
+                    }}
+                    disabled={isParsingWithProgress || autofillLoading}
+                  >
+                    <Sparkles className="h-4 w-4" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)